Tutorials/Python_Developer_Tutorials/foundation/Commanding-Remote-RF/Readme.md

```diff
@@ -1,89 +1,100 @@
-
-> NOTE: Ideally the proxy assigned callsign/ID matches the unit device configuration but this is not controlled or required and care should be taken.
+> NOTE: Ideally the proxy assigned callsign/ID matches the unit device configuration but this is not controlled or required and care should be taken to ensure the information is correctly configured.
 
 ```python
-#Local device information (Must match proxy assigned information)
-local_device_callsign = 'kb1lqd'  #case independent
-local_device_node_id = 1
+[DEVICES]
+UNITS=2
+
+; Transmitter - This should match the connected Faraday unit as assigned in Proxy configuration
+UNIT0CALL=REPLACEME
+UNIT0ID= REPLACEME
 
-#Remote device information (Must match unit programming)
-remote_callsign = 'kb1lqd'  #case independent
-remote_id = 2
+; Receiver - This should match the programmed callsign of the remote Faraday device to be commanded (receive)
+UNIT1CALL=REPLACEME
+UNIT1ID= REPLACEME
 ```
 
 
 ## Execute Tutorial Script
 
-While running the tutorial script you should see the green led (LED #1) light up on the remote unit and the red LED flashing on the local unit. Default Faraday operation enables the red LED whenever RF transmissions are occuring.
-
-The tutorial script terminal output during a succesful operation displays raw command packets as transmitted for reference.
+While running the tutorial script you should see the green led (LED #1) and red LED (LED #2) light up on the remote unit. During the actuation of DIGITAL_IO_0 you will measure 3.3V and 0V respectively for ON and OFF commands.
 
 ![Successful Operation Terminal](Images/Output_Example_Success.png "Successful Operation Terminal")
 
 # RF Commanding Design Summary
 
-The command functionality on Faraday is operating in an application running on Faraday itself and parses a command application packet protocol to determine actions needed. This is common between both local and RF commands however sending and RF command simply encapsulates a local command (for the remote unit) within a local "transmit RF command" command (packet). This results in the remote device receiving a "local" command from a remote device, it knows no difference.
+The command functionality on Faraday is operating in an application running on Faraday itself and parses command application packets to determine actions needed. This is common between both local and RF commands however sending and RF command simply encapsulates a local command (for the remote unit) within a wireless packet transmission both address to the intended device and respective command application "port". This results in the remote device receiving a "local" command from a remote device, it knows no difference.
```

Comment:
"packet transmission both address to the intended device" should this be addressed?